2_Growth-Maturity-Decline.md

```diff
@@ -1,15 +1,33 @@
 # Metric: Growth-Maturity-Decline
 
 
+## Issue Resolution
  Activity Metric | Description
  --- | ---
- Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
- Bug Time to Close | Time from bug opened to bug closed over time
- Time to Merge | Time from submission to merge or abandon over time
- Review Efficiency | Number of merged patches / Number of abandoned patches over time
- Number of New Contributors | Number of first time contributors over time
- Community Growth | Number of new developers over time
- Community Toxicity | Toxic behavior and language over time
+ Open issues | Total number of open issues.
+ Closed issues | Total number of closed issues.
+ Resolution efficiency | Number of closed issues / number of abandoned issues.
+ Average & median time of open issues | The average & median amount of time issues remain open.
+ Average & median time of first response | The average & median amount of time until the first response to an issue is made.
+ Distribution of open issues over time | Proportional frequency of open issue time duration.
+ Distribution of issue resolution over time | Proportional frequency of closed issue time duration.
+
+## Code Development
+ Activity Metric | Description
+ --- | ---
+ Number of commits | Total number of commits.
+ Number of lines changed | Total number of lines of code that have been changed.
+ Average & median time to merge code | The average & median amount of time it takes to get code merged upstream. 
+ Distribution of time to merge | Proportional frequency of code merge to upstream time duration.
+ Review efficiency | Number of merged patches / number of abandoned patches.
+
+## Project Communication
+ Activity Metric | Description
+ --- | ---
+ Total communications | Total number of posts made to communication platforms.
+ Average & median communication response time | The average & median time taken for communications to be responded to.

```

Comment:
We have been hearing that responses are good, but not always helpful. For example a message of type "+1" does not help. A solution we have been hearing is to look specifically for time taken for a response from a maintainer. Would that change still get to the core idea behind this criteria?

Reply:
Agreed, this sounds better: Average & median time for first maintainer response.

Imo, this solves two problems, 1) it focuses on responses that have meaning, and 2) it doesn't leave the door open after the final response in a thread (i.e. there won't be a way to judge if a conversation is "done" so the clock would keep ticking after the last response to a thread and would make this number meaningless).